game-runner/game-runner-config.json

```diff
@@ -5,7 +5,7 @@
   "verbose-mode": true,
   "max-runtime-ms": 1000,
   "player-a": "../starter-bots/javascript",
```

Comment:
What does the readme mention for the required software at first?
This way, participants need to install Java and NodeJS

 "player-a": "../starter-bots/java", might make the installation easier for them

Reply:
It doesn't and I agree. But the reference bot does not exist which is javascript. So the build and runner does not work! This is just a fix to get it working.

Reply:
I have changed more than this to get things working in other areas. I will be submitting the changes in another pull request. Just what is provided does not work out of the box, for instance the script to make the starter package wants gradle, but it is scala and no gradle info in the folder.